### PR TITLE
local development improvements

### PR DESCRIPTION
**Pull request by dagbay-rh**
## Summary
Some changes I made while working on this service locally, for QOL improvements
* updated config to set default cert locations so we dont have to do this in the makefile over and over
* use config key where possible
* add makefile options to use build executable separate from using `go run` 
* added some error messages
* removed usage of `modd`, seems like extra fluff we don't need
* removed travis build
* updated jenkins build to run our unit tests so we don't need travis build
* updated readme to reflect changes and add some info about the service

# Platform Security

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist 

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: bundle_sync/main.go [hunk 1]
```diff
@@ -116,8 +117,8 @@ func main() {
 		skus := make(map[string][]string)
 		current_skus := make(map[string][]string)
 		url := fmt.Sprintf("%s%s%s",
-			options.GetString("SUBS_HOST"),
-			options.GetString("SUB_API_BASE_PATH"),
+			options.GetString(config.Keys.SubsHost),
```
Comment (petracihalova) [inline bundle_sync/main.go:120]: can we have the same for the code above in the line 93?

**Review by petracihalova — CHANGES_REQUESTED** [review]
nice work, just tiny comments 👍 😇 ❤️

**Review by Ellen-Yi-Dong — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Entitlements Service
 
+Entitlements service serves as a proxy to various backend redhat IT services. It performs the following functions:
```
[the remark this answers is not in the document]
Comment (dagbay-rh) [reply README.md:3]: done and done!

File: bundle_sync/main.go (hunk 1 above)
Comment (dagbay-rh) [reply bundle_sync/main.go:120]: ah yes, somehow didn't notice that haha

**Review by Ellen-Yi-Dong — APPROVED** [review]
LGTM!

**Review by petracihalova — APPROVED** [review]
LGTM 👍

**Pull request merged**